cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -26,6 +26,7 @@
 public final class Constants {
 
   public static final String ARCHIVE_DIR = "archive";
+  public static final String CFG_ROOT_NAMESPACE = "root.namespace";
```

Comment:
The reason I had it like that, is because the value of the variable is actually the key in the CConfiguration (as opposed to the value in the CConfiguration).

Same is used in other variables in Constants.java (prefixed by CFG if its the key in the CConfiguration).